src/tests/component/Auth.test.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Auth from '../../components/Auth/Auth';
+
+describe('<Auth />', () => {
+  test('should render auth', () => {
+    const wrapper = shallow(<Auth />);
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  test('should render auth', () => {
```

Comment:
Line 16 has the same test description

Reply:
Oh, thanks. I think most lines do have the same description, Will fix this shortly